api/envoy/extensions/filters/http/fault/v3/fault.proto

```diff
@@ -141,4 +141,6 @@ message HTTPFault {
   // The runtime key to override the :ref:`default <config_http_filters_fault_injection_runtime>`
   // runtime. The default is: fault.http.abort.grpc_status
   string abort_grpc_status_runtime = 14;
+
+  bool disable_downstream_server_tracing = 15;
```

Comment:
Please add comment explaining what this field does. This info goes into envoy doc.
Also this is not really tracing related. Perhaps name the field `disable_dynamic_stats` or `disable_downstream_cluster_stats`?

Reply:
Thanks, will fix.

Reply:
Also please mention the default value in the comment.